### PR TITLE
crush: update tunable docs.  change default profile to jewel.

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: doc/rados/operations/crush-map.rst [hunk 1]
```diff
@@ -947,33 +947,34 @@ The following example removes the ``rack12`` bucket from the hierarchy::
...
-reflect support for tunables.
-
-If the OSDMap currently used by the ``ceph-mon`` or ``ceph-osd``
-daemon has non-legacy values, it will require the ``CRUSH_TUNABLES`` or ``CRUSH_TUNABLES2``
-feature bits from clients and daemons who connect to it.  This means
-that old clients will not be able to connect.
+Over time, we have made (and continue to make) improvements to the
+CRUSH algorithm used to calculate the placement of data.  In order to
+support the change in behavior, we have introduced a series of tunable
+options that control whether the legacy or improved variatio of the
```
Comment (xiexingguo) [inline doc/rados/operations/crush-map.rst]: Do you mean "variation"?

File: doc/rados/operations/crush-map.rst [hunk 2]
```diff
@@ -1017,41 +1026,82 @@ CRUSH_TUNABLES3
...
+
+ * There is a new bucket type (``straw2``) supported.  The new
+   ``straw2`` bucket type fixes several limitations in the original
+   ``straw`` bucket.  Specifically, the old ``straw`` buckets would
+   change some mappings that should have changed when a weight was
+   adjusted, while ``straw2`` achieves the original goal of only
+   changing mappings to or from the bucket item whose weight has
+   changed.
+
+ * ``straw2`` is the default for any newly created bucketes.
```
Comment (xiexingguo) [inline doc/rados/operations/crush-map.rst]: typo : bucketes-> buckets

Comment (liewegas) [inline doc/rados/operations/crush-map.rst]: fixed, thanks!

**Comment by liewegas** [comment]
@jecluis @athanatos will review

**Comment by athanatos** [comment]
Ah, the min check relies on the fact that we've been choosing version names such that newer ones sort higher.

lgtm

**Comment by gregsfortytwo** [comment]
Make check gitbuilder has been failing since this PR merged (http://gitbuilder.sepia.ceph.com/gitbuilder-ceph-tarball-trusty-amd64-basic/log.cgi?log=203d604d7221ac3bb721a39efff72f2f0dbb1193) — it looks to me like it's just expected output changes as a result of changing the CRUSH defaults, but definitely needs to be updated!

**Comment by tchaikov** [comment]
@gregsfortytwo seems sage has already a PR (#8107) addressing this.

**Pull request merged**